crates/pera-types/src/pera_system_state/epoch_start_pera_system_state.rs

```diff
@@ -100,9 +103,14 @@ pub struct EpochStartSystemStateV1 {
     epoch_start_timestamp_ms: u64,
     epoch_duration_ms: u64,
     active_validators: Vec<EpochStartValidatorInfoV1>,
+    encrypted_decryption_key_shares: Option<Vec<Vec<u8>>>,
 }
 
 impl EpochStartSystemStateV1 {
+    pub fn get_encrypted_decryption_key_shares(&self) -> Option<Vec<Vec<u8>>> {
+        self.encrypted_decryption_key_shares.clone()
+    }
+
```

Comment:
why is this better than accessing the field directly?

Reply:
Simply because all the other fields here are also private, and we need a way to access them.
Additionally, it makes sense that the data within the start state is initialized once and has no further access.